chroma-manager.conf.template

```diff
@@ -431,6 +431,8 @@ server {
         proxy_set_header Http-Host $http_host;
         proxy_pass {{UPDATE_HANDLER_PROXY_PASS}};
     }
+
+    {{INCLUDES}}
 }
```

Comment:
Could be `include /etc/nginx/chroma-manager.d/*.conf` so we could add extra configuration just by dropping in more files?

Reply:
I think we can follow this up in a later patch.